Sources/Apodini/Request/PathParameter.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+
+/// A `@PathComponent` can be used in `Component`s to indicate that a part of a path is a parameter and can be read out in a `Handler`
+@propertyWrapper
+public struct PathParameter<Element> {
```

Comment:
the `@PathParameter` still needs to conform somehow to the `PathComponent` protocol.
Otherwise it can't be used as stated in the example:

```swift
// ...
struct TestWebService: WebService, Component {
    @PathParameter var birdID: Bird.ID

    var content: some Component {
        Group("api", "birds", /* >>> */birdID /* <<< */) {
            ExampleBirdComponent(birdID: $birdID)
            Group("nests") {
                ExampleNestComponent(birdID: $birdID)
            }
        }
    }
}
// ...
```

Reply:
A quick follow up. Wouldn't it make sense that the generic `Element` must conform to `LosslessStringConvertible`.
Thus we may need to create our own version of `Identifiable` where the associated type `ID` also conforms for `LosslessStringConvertible` (instead of `Hashable`).

Reply:
Yes, you are right, it needs to conform to `PathComponent`, I will update this.
Element in general must not need to conform to `LosslessStringConvertible` but we should add a precondition check that this is the case in case you use `.http(.path)` option.